### PR TITLE
created a header, left nav and section

### PR DESCRIPTION
**Pull request by ChidebeLawrence**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,39 @@
+import logo from './assets/images/logo.png'
+import profile_img from './assets/images/profile_img.png'
+import { BiChevronDown } from 'react-icons/bi'
+import { IoChevronBackOutline } from 'react-icons/io5'
+
 function App() {
-  return <h1 className="bg-yellow-500 text-xl">hello world</h1>
+  return (
```
Comment (Irotochukwusamuel) [inline src/App.tsx:6]: please don't create any Tsx inside here. have a different component that handles this. ensure you have a good naming convention.

File: package.json [hunk 2]
```diff
@@ -19,6 +19,8 @@
     "eslint-import-resolver-typescript": "^3.5.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icon": "^1.0.0",
```
Comment (Irotochukwusamuel) [inline package.json]: remove the incorrect one

File: src/App.tsx (hunk 1 above)
Comment (ChidebeLawrence) [reply src/App.tsx:6]: resolved

**Comment by ChidebeLawrence** [comment]
resolved

**Pull request closed without merging**